lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -3130,6 +3129,36 @@ void PatchInOutImportExport::patchFsBuiltInOutputExport(Value *output, unsigned
 void PatchInOutImportExport::patchCopyShaderGenericOutputExport(Value *output, unsigned location,
                                                                 Instruction *insertPos) {
   addExportInstForGenericOutput(output, location, 0, insertPos);
+
+  // Generate "exp" instruction here instead in visitReturnInst where the copyshader stage will not create "exp"
+  // instruction to export the generic output if the xfb is enabled.
+  if (m_pipelineState->getShaderResourceUsage(m_shaderStage)->inOutUsage.enableXfb) {
+    BuilderBase builder(*m_context);
+    builder.SetInsertPoint(insertPos);
+    auto undef = UndefValue::get(Type::getFloatTy(*m_context));
+    SmallVector<Value *, 8> attribExport(8, undef);
```

Comment:
nit: use the default SmallVector size: https://llvm.org/docs/ProgrammersManual.html#llvm-adt-smallvector-h